gaek/ndb_json.py

```diff
@@ -67,14 +67,25 @@ def encode_generator(obj):
   return list(obj)
 
 
-def encode_key(obj):
+def encode_key_as_entity(obj):
   """Get the Entity from the ndb.Key for further encoding."""
   # NOTE(erichiggins): Potentially poor performance for Models w/ many KeyProperty properties.
+  # NOTE(ronufryk): Potentially can cause circular references and "RuntimeError: maximum recursion depth exceeded"
```

Comment:
I will try it with collect data script, once I'm back to work. I will add this information then.